ggircs-app/app/components/Dashboard/SwrsDataAccessCard.tsx

```diff
@@ -1,22 +1,16 @@
 import React from "react";
-import { Card, ListGroup } from "react-bootstrap";
+import Card from "@button-inc/bcgov-theme/Card";
+import Button from "@button-inc/bcgov-theme/Button";
 import Link from "next/link";
 
 const SwrsDataAccessCard: React.FunctionComponent = () => (
-  <Card>
-    <Card.Body>
-      <Card.Title>SWRS</Card.Title>
-      <Card.Text>
-        Access reports and attachments from the Single Window Reporting System
-      </Card.Text>
-    </Card.Body>
-    <ListGroup variant="flush">
-      <ListGroup.Item>
-        <Link href="/swrs-browser" passHref>
-          <Card.Link href="#">Browse SWRS files</Card.Link>
-        </Link>
-      </ListGroup.Item>
-    </ListGroup>
+  <Card title="SWRS">
+    Access reports and attachments from the Single Window Reporting System
+    <br />
+    <br />
```

Comment:
No using `br` to create whitespace. We should be able to use `styled-jsx`?

Reply:
I figured this is what the service toolkit recommends...I don't have any opinions :)

Reply:
`br` should only be used for a line break within a block of text. For spacing, using `p` elements and/or `margin` properties is the recommended approach.
 I'll open an issue in the toolkit repo so that the example is updated to follow best practices